diff --git a/oneforall.py b/oneforall.py
index 3d62fe4..7b3ea7e 100644
--- a/oneforall.py
+++ b/oneforall.py
@@ -32,7 +32,7 @@ blue = '\033[01;34m'
 red = '\033[1;31m'
 end = '\033[0m'
 
-version = 'v0.4.5'
+version = 'v0.4.6'
 message = white + '{' + red + version + ' #dev' + white + '}'
 
 oneforall_banner = f"""
diff --git a/requirements.txt b/requirements.txt
index 42460b7..b393348 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -4,7 +4,7 @@ certifi==2022.06.15
 chardet==5.0.0
 colorama==0.4.4
 dnspython==2.2.1
-exrex==0.10.5
+exrex==0.11.0
 fire==0.4.0
 future==0.18.2
 idna==3.3
