diff --git a/setup.py b/setup.py
index 110dda9..9bea371 100755
--- a/setup.py
+++ b/setup.py
@@ -6,7 +6,7 @@ with open("README.md", "r", encoding = "utf-8") as fh:
 
 setuptools.setup(
     name = "smbmap",
-    version = "1.9.1",
+    version = "1.9.2",
     author = "ShawnDEvans",
     author_email = "Shawn.Evans@knowledgeCG.com",
     description = " SMBMap is a handy SMB enumeration tool ",
@@ -26,7 +26,7 @@ setuptools.setup(
     install_requires = [
         'impacket',
         'pyasn1',
-        'pycrypto',
+        'pycryptodome',
         'configparser',
         'termcolor',
     ],
